custom/dhis2/tasks.py

```diff
@@ -214,28 +236,22 @@ def gen_children_only_ours(domain):
             yield CommCareCase.wrap(doc)
 
 
-# TODO: Use case forwarding, or form forwarding of registration forms
-# @periodic_task(run_every=timedelta(minutes=5))  # Run every 5 minutes to match forwarded forms
-def sync_cases():
+@periodic_task(run_every=crontab(minute=2, hour=2))  # Run daily at 02h02
+def fetch_cases():
     """
     Create new child cases in CommCare for nutrition tracking, and associate
     CommCare child cases with DHIS2 child entities and enroll them in the
     Pediatric Nutrition Assessment and Underlying Risk Assessment programs.
     """
-    for domain in Domain.get_all():
-        settings = Dhis2Settings.for_domain(domain.name)
-        if settings is None or not settings.is_enabled():
-            continue
-
+    for settings in Dhis2Settings.all_enabled():
+        logger.info('DHIS2: Fetching cases for domain "%s" from "%s"', settings.domain, settings.dhis2['host'])
         children = get_children_only_theirs(settings)
         pull_child_entities(settings, children)
 
-        children = gen_children_only_ours(domain.name)
-        push_child_entities(settings, children)
-
 
+# There is a large number of org units, but the lookup table is not deployed to handsets.
 @periodic_task(run_every=crontab(minute=3, hour=3))  # Run daily at 03h03
-def sync_org_units():
+def fetch_org_units():
```

Comment:
Oh right, I see there is a setting to say if the fixture is 'visible to all
users'